cluster_generator/codes.py

```diff
@@ -27,8 +38,9 @@ def write_amr_particles(particles, output_filename, ptypes,
     from scipy.io import FortranFile
 
     if Path(output_filename).exists() and not overwrite:
-        raise IOError(f"Cannot create {output_filename}. "
-                      f"It exists and overwrite=False.")
+        raise IOError(
+            f"Cannot create {output_filename}. " f"It exists and overwrite=False."
```

Comment:
These strings should now be joined together by hand.

Reply:
Fixed all of these instances.